.github/workflows/kind_multicluster_e2e_tests.yaml

```diff
@@ -129,9 +146,15 @@ jobs:
         with:
           name: k8ssandra-operator
           path: /tmp
+      - name: Download k8ssandra-secrets-webhook image
```

Comment:
My main note from my preliminary review is that there is an issue with the kuttl tests, I think that you'll need to add the same logic there as well as to the other test workflow files (especially `.github/workflows/kuttl_tests.yaml`, `.github/workflows/version_tests.yaml`).

The part that needs to be added is around line 100 and 94 respectively, and I've reproduced it for clarity below:
```
     - name: Download k8ssandra-secrets-webhook image
        uses: actions/download-artifact@v2
        with:
          name: k8ssandra-secrets-webhook
          path: /tmp
      - name: Load images
        run: |
          docker load --input /tmp/k8ssandra-operator.tar
          docker load --input /tmp/k8ssandra-secrets-webhook.tar
```

Hopefully that will fix the bulk of the test failures.